lib/Processor/DefaultProcessor.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Productsup\Flexilog\Processor;
+
+/**
+ * Default Processor
+ */
+class DefaultProcessor extends AbstractProcessor implements ProcessorInterface
+{
+    /**
+     * Interpolates context values into the message placeholders.
+     *
+     * @param string $message Message to Log with Placeholders, defined by {curly}-braces.
+     * @param array  $context Key/Value array with properties for the Placeholders.
+     *
+     * @return string $message Message with Placeholders replaced by the context.
+     */
+    public function interpolate($message, array $context = array())
+    {
+        // build a replacement array with braces around the context keys
+        $replace = array();
+        foreach ($context as $key => $val) {
```

Comment:
Maybe context preparation should take place automatically inside this method? Or indicate that `$context` should actually be a _preparedContext_, since that handles most data types.